package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "token",
-  "version": "0.1.0",
+  "version": "0.2.0",
```

Comment:
I think the version 0.1.1 is better to this case, It's a small change that do not break in nothing the code interface, it's in case of follow the [semantic version pattern](https://semver.org/).